easybuild/tools/utilities.py

```diff
@@ -99,5 +102,9 @@ def import_available_modules(namespace):
                 mod_name = module.split(os.path.sep)[-1].split('.')[0]
                 modpath = '.'.join([namespace, mod_name])
                 _log.debug("importing module %s" % modpath)
-                modules.append(__import__(modpath, globals(), locals(), ['']))
+                try:
+                    mod = __import__(modpath, globals(), locals(), [''])
+                except ImportError as err:
```

Comment:
Not all over the place, just on the very top, in your binary, where you can handle things. and print output to the user.
if it goes wrong, it will be very wrong, and bubble up all to the top.

If somewhere in that path up to the top you have a `except EasyBuildError`, which can happen, because it's starting to look like everything that can fail starts to raise EasyBuildErrors, so if I call a function and can recover from certain failures I have to catch EasyBuildError, it won't stop and continue in a weird state.

And how do you know that something is very wrong and you're done? You're writing a library here, you have no idea of where it will end up being used.
If you want it to be generic and reusable, then you don't want to mask errors, and you can't assume every error is fatal.

if you're not catching this error, it will be raised aswell, so things will stop, no need to catch it and mask it with another error?